cfme/tests/openstack/cloud/test_volume_backup.py

```diff
@@ -64,6 +67,60 @@ def incremental_backup(volume_backup, provider):
         logger.warning('Exception during volume backup deletion - skipping..')
 
 
+@pytest.fixture(scope='function')
+def new_instance(provider):
+    prov_data = provider.data['provisioning']
```

Comment:
Consider creating the VM via rest, or on the provider itself and do a provider refresh, as this adds a lot of time to the test creating the vm via a CFME request. Its not really necessary for the test to do this setup via UI.